### ISSUE TITLE
Error: Browser-only verison of superagent could not find XHR

### ISSUE THREAD

**Issue by devhyunjae** [filestack/filestack-js#39]
Hi,

Trying to use `filestack-js` in `node`, and have this error

Error
```Error: Browser-only verison of superagent could not find XHR```

Code
```
const filestack = require('filestack-js')
const instance = filestack.init('KEY')
instance.storeURL('https://d1wtqaffaaj63z.cloudfront.net/images/NY_199_E_of_Hammertown_2014.jpg').then(res => console.log(res)).catch(err => console.log(err))
```

```
> Error: Browser-only verison of superagent could not find XHR
    at Function.request.getXHR (node_modules/filestack-js/dist/filestack.js:2:36458)
    at Request._end (node_modules/filestack-js/dist/filestack.js:2:43272)
    at Request.end (node_modules/filestack-js/dist/filestack.js:2:43191)
    at /node_modules/filestack-js/dist/filestack.js:2:75362
    at storeURL /node_modules/filestack-js/dist/filestack.js:2:75231)
    at Object.storeURL$$1 [as storeURL] (=node_modules/filestack-js/dist/filestack.js:2:124066)
    at Object.storeURL (/node_modules/filestack-js/dist/filestack.js:2:127526)
    at repl:1:10
    at ContextifyScript.Script.runInThisContext (vm.js:23:33)
    at REPLServer.defaultEval (repl.js:339:29)
```

**Comment by velveteer** [comment]
Node support is not there yet, but we are working on a solution for Node environments. Whether that ends up in this SDK or in a separate module is still under consideration.

**Comment by olalonde** [comment]
@velveteer any update on this? I'd be happy to send a PR but you are using some private dependencies that are not on Github? (e.g. `api-client`)

**Comment by velveteer** [comment]
This should be resolved in 1.0.0-rc1. Please see the release notes here: https://github.com/filestack/filestack-js/issues/101. 

If you feel this issue should be reopened please let us know.
